_packages/@karrotmarket/gatsby-plugin-lokalise-translation/gatsby/gatsby-node.ts

```diff
@@ -1,126 +1,82 @@
-    limit: 500,
-    include_translations: 1,
-  });
-
-  const keys = items as Key[];
-
-  if (!keys || keys.length === 0) {
-    reporter.panic(`no keys found at project_id=${lokaliseProjectId}`);
+}) => {
+  if (!(
+    node.internal.type === 'File' &&
+    node.internal.mediaType === 'application/json' &&
+    node.sourceInstanceName === 'translations'
+  )) {
+    return;
   }
 
+  const { program } = store.getState();
+  const baseDir = program.directory;
+
   type TranslationKey = string;
   type TranslationMessage = string;
-  type LokaliseTranslation = Record<TranslationKey, TranslationMessage>;
-
-  type Locale = string;
-  type LocalizedLokaliseTranslation = Record<Locale, LokaliseTranslation>;
-
-  const aggregatedTranslation: LocalizedLokaliseTranslation = {};
-
-  for (const key of keys) {
-    if (!Array.isArray(key.translations)) {
-      continue;
-    }
-    const keyName = (key.key_name as Record<string, string>)['web'];
-    for (const t of key.translations as Translation[]) {
-      if (!aggregatedTranslation[t.language_iso]) {
-        aggregatedTranslation[t.language_iso] = {};
-      }
-      aggregatedTranslation[t.language_iso][keyName] = t.translation;
-    }
-  }
-
-  let emitted = false;
-
-  for (const [locale, messages] of Object.entries(aggregatedTranslation)) {
-    const keys = Object.keys(messages);
-
-    if (!emitted) {
-      await renderMessagesFragment(
-        keys,
-        reporter.stripIndent,
-      );
-      await renderMessagesType(
-        keys,
-        reporter.stripIndent,
-      );
-      emitted = true;
-    }
+  type LokaliseMessages = Record<TranslationKey, TranslationMessage>;
+
+  const messages: LokaliseMessages = JSON.parse(await loadNodeContent(node));
+  const translationNode: NodeInput = {
+    id: createNodeId(`${node.id} >>> LokaliseTranslation`),
+    parent: node.id,
+    internal: {
+      type: 'LokaliseTranslation',
+      contentDigest: createContentDigest(messages),
+    },
+    messages,
+  };
+
+  actions.createNode(translationNode);
+  actions.createParentChildLink({
+    parent: node,
+    child: translationNode,
+  });
 
-    const content = {
-      locale,
-      messages,
-    };
-
-    actions.createNode({
-      id: createNodeId(`LokaliseTranslation:${locale}`),
-      ...content,
-      internal: {
-        type: 'LokaliseTranslation',
-        contentDigest: createContentDigest(content),
-      },
-    });
+  if (!emitted) {
+    await renderMessagesFragment(
+      path.join(baseDir, 'src/__generated__'),
+      Object.keys(messages),
+      reporter.stripIndent,
+    );
+    emitted = true;
   }
 };
 
 async function renderMessagesFragment(
+  outputDir: string,
   keys: string[],
   stripIndent = String.raw,
 ) {
-  const outputDir = path.join(__dirname, 'src');
-  fs.mkdir(outputDir, { recursive: true });
-
-  const outputPath = path.join(outputDir, 'fragments.js');
+  const outputPath = path.join(outputDir, 'lokalise-translation.js');
```

Comment:
outDir 이 없어서 터지는거 ㅅ같네요

```suggestion
  const outputPath = path.join(outputDir, 'lokalise-translation.js');
  await fs.mkdir(outputDir, { recursive: true });
```